packages/rancher-sriov/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,52 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,23 +1,37 @@
+ operator:
+   tolerations:
+-    - key: "node-role.kubernetes.io/master"
+-      operator: "Exists"
+-      effect: "NoSchedule"
++  - effect: NoSchedule
++    key: node-role.kubernetes.io/master
++    operator: Exists
++  - effect: NoSchedule
++    key: cattle.io/os
++    operator: Equal
++    value: linux
+   nodeSelector:
+-    node-role.kubernetes.io/master: ""
++    node-role.kubernetes.io/master: "true"
+   nameOverride: ""
+   fullnameOverride: ""
+-  resourcePrefix: "openshift.io"
++  resourcePrefix: "rancher.io"
```

Comment:
What are the implications of changing the resourcePrefix? Can we safely do this without running into any other issues (e.g. CRDs still have openshift.io on them right?)

Reply:
Yes, this is safe. CRDs still have the openshift.io suffix. The sriov resources are named $resourcePrefix/$nic_model, e.g. rancher.io/intel